### PR TITLE
Remove the invocation of wakuext_chatMentionReplaceWithPublicKey

### PR DESCRIPTION
**Pull request by qfrank**
#### Description
RT.
Relate status-go [PR](https://github.com/status-im/status-go/pull/4579)

#### Platforms

- Android
- iOS

#### Areas that maybe impacted
chat message with mention should be reolved correctly as before.

### Steps to test

- UserA Login Status
- Invite UserB to join community
- After UserB join community, send a message with mention in community channel, e.g. "hello @UserB"
- Check the message rendering on both side, we should expect: the mention in the message should be recognized correctly with highlight as before

status: ready

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/contexts/chat/messenger/composer/events.cljs [hunk 1]
```diff
@@ -207,24 +207,13 @@
...
-  [{:keys [db] :as cofx} current-chat-id editing-message input-text new-text]
-  (rf/merge cofx
-            (if editing-message
-              (send-edited-message new-text editing-message)
-              (send-messages new-text current-chat-id))))
+  {:events [:chat.ui/send-current-message]}
+  [{{:keys [current-chat-id] :as db} :db :as cofx}]
+  (let [{:keys [input-text metadata]} (get-in db [:chat/inputs current-chat-id])
+        editing-message               (:editing-message metadata)]
+    (rf/merge cofx
```
Comment (flexsurfer) [inline src/status_im/contexts/chat/messenger/composer/events.cljs]: redundant `(rf/merge)` could be just
```
(if editing-message
                (send-edited-message cofx input-text editing-message)
                (send-messages cofx input-text current-chat-id))
```

**Comment by pavloburykh** [comment]
@qfrank thanks for the PR! Could you please rebase status-go and mobile branch before we continue with testing? thank you!

**Comment by qfrank** [comment]
> @qfrank thanks for the PR! Could you please rebase status-go and mobile branch before we continue with testing? thank you!

done @pavloburykh

**Comment by pavloburykh** [comment]
@qfrank thanks for the PR. No issues from my side. Ready for merge.

**Pull request merged**